wdl/transforms/draft3/src/main/scala/wdl/draft3/transforms/linking/graph/package.scala

```diff
@@ -48,17 +50,36 @@ package object graph {
     }
   }
 
+  implicit val callElementUnlinkedValueGenerator: UnlinkedValueGenerator[CallElement] = new UnlinkedValueGenerator[CallElement] {
+    override def generatedValueHandles(a: CallElement, typeAliases: Map[String, WomType]): ErrorOr[Set[GeneratedValueHandle]] = {
+      // TODO this will need to be filled in for calls with outputs
+      Set[GeneratedValueHandle]().empty.valid
+    }
+  }
+
   implicit val graphElementUnlinkedValueConsumer: GraphElementValueConsumer[WorkflowGraphElement] = new GraphElementValueConsumer[WorkflowGraphElement] {
     override def graphElementConsumedValueHooks(a: WorkflowGraphElement, typeAliases: Map[String, WomType]): ErrorOr[Set[UnlinkedConsumedValueHook]] = a match {
       case InputDeclarationElement(_, _, None) => Set.empty[UnlinkedConsumedValueHook].validNel
       case DeclarationElement(_, _, Some(expr)) => expr.expressionConsumedValueHooks.validNel
       case a: ScatterElement => a.graphElementConsumedValueHooks(typeAliases)
       case a: IfElement => a.graphElementConsumedValueHooks(typeAliases)
+      case a: CallElement => a.graphElementConsumedValueHooks(typeAliases)
       // TODO fill in other expression types
       case other => throw new Exception(s"Cannot generate consumed values for WorkflowGraphNodeElement $other")
     }
   }
 
+  implicit val callElementUnlinkedValueConsumer: GraphElementValueConsumer[CallElement] = new GraphElementValueConsumer[CallElement] {
+    override def graphElementConsumedValueHooks(a: CallElement, typeAliases: Map[String, WomType]): ErrorOr[Set[UnlinkedConsumedValueHook]] = {
+      a.body match {
+        case Some(callBodyElement: CallBodyElement) => callBodyElement.inputs.toSet.flatten { input: ExpressionElement.KvPair =>
```

Comment:
I suspect that a `traverse` followed by a `.map(_.flatten)`  is better here. I think that'll mean we get all of the errors if there's more than one